wheatley/bot.py

```diff
@@ -106,16 +106,16 @@ def log_invalid_key(message):
     def _on_row_gen_change(self, row_gen_json):
         try:
             self.next_row_generator = json_to_row_generator(row_gen_json, self.logger)
+            self.next_row_generator.set_tower_size(self._tower.number_of_bells)
 
             self.logger.info("Successfully updated next row gen")
         except RowGenParseError as e:
             self.logger.warning(e)
 
     def _on_size_change(self):
-        if not self.row_generator.is_tower_size_valid(self._tower.number_of_bells):
-            self.logger.warning(f"Row generation requires {self.row_generator.number_of_bells} \
-bells, but the current tower has {self._tower.number_of_bells}.  Wheatley will crash when you go \
-into changes unless something is done!")
```

Comment:
Surely this warning is still useful, but should only trigger if we're trying to ring a method/comp that has too many bells?

Reply:
Oops I see this has moved elsewhere